doc/README.md

```diff
@@ -1,37 +1,16 @@
-Merit Core
+Merit
 =============
 
 Setup
 ---------------------
-Merit Core is the original Merit client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Merit transactions (which is currently more than 100 GBs); depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
-
-To download Merit Core, visit [meritcore.org](https://meritcore.org/en/releases/).
-
-Running
----------------------
-The following are some helpful notes on how to run Merit on your native platform.
-
-### Unix
-
-Unpack the files into a directory and run:
-
-- `bin/merit-qt` (GUI) or
-- `bin/meritd` (headless)
-
-### Windows
-
-Unpack the files into a directory, and then run merit-qt.exe.
-
-### OS X
-
-Drag Merit-Core to your applications folder, and then run Merit-Core.
+Merit's mobile wallet is the easiest way to get started.
```

Comment:
These docs are about meritd and merit core. Presumably the audience is interested in using meritd instead of the mobile wallet if they are reading this doc. I would not direct them to the light wallet but explane how to get the code, build it, and execute. If we provide binaries later we can add references of how to get pre-compiled versions.